clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -164,18 +164,18 @@ Decl *SemaHLSL::ActOnStartBuffer(Scope *BufferScope, bool CBuffer,
   return Result;
 }
 
-// Calculate the size of a legacy cbuffer type based on
+// Calculate the size of a legacy cbuffer type in bytes based on
```

Comment:
Cbuffer & packoffset layout calculations have been using 3 different units - bits, bytes, and 4-byte chunks. I am changing all to use bytes.